ArduPlane/takeoff.cpp

```diff
@@ -305,3 +305,29 @@ void Plane::landing_gear_update(void)
     g2.landing_gear.update(relative_ground_altitude(g.rangefinder_landing));
 }
 #endif
+
+/*
+ check takeoff_timeout; checks time after the takeoff start time; returns true if timeout has occurred and disarms on timeout
+*/
+
+bool Plane::check_takeoff_timeout(void)
```

Comment:
Should be a method on `Mode`

Reply:
That is true, but so should the take off state. I don't think we need to do it in this PR.